### DISCUSSION TITLE
How to create a new v3 project from scratch? (Improve docs?)

### DISCUSSION THREAD

**Discussion by fdcastel** [chakra-ui/chakra-ui#9221] [Q&A] [answered]
I’m attempting to create a V3 project from scratch using Vite, but I’ve encountered several challenges. 

The [current documentation](https://www.chakra-ui.com/docs/get-started/frameworks/vite) directs users to a [subfolder within the main project](https://github.com/chakra-ui/chakra-ui/tree/main/sandbox/vite-ts) repository. This required me to clone the entire repository just to access the necessary files. 

Unfortunately, even after doing so, the setup fails with an error related to [this ](https://github.com/chakra-ui/chakra-ui/blob/main/sandbox/vite-ts/package.json#L11) line (`workspace:*` ?).

Ironically, the most helpful resource I found was indirectly provided when attempting to open an issue regarding this problem. A required field in the issue submission form linked to a [StackBlitz template](https://stackblitz.com/edit/chakra-ui-v3?file=src%2FApp.tsx), which appears to function correctly.

This could present a challenge for new users, potentially discouraging them from investing the time needed to explore all the great features of Chakra and prompting them to consider alternative solutions.

**Comment by segunadebayo** [comment, marked as answer]
I'd recommend watching this video from Esther on our team.
https://www.youtube.com/watch?v=kZOGVlwm0_M&list=PLx2Y9Sna27Xtmbo3CAbHvQJFU064zB4kF&index=2&ab_channel=Esther

We would consider moving the template to their dedicated repo.
